lib/ukrand/swrand.c

```diff
@@ -32,23 +32,21 @@
 #include <uk/config.h>
 #include <uk/print.h>
 #include <uk/init.h>
+#include <uk/assert.h>
+#include <uk/arch/random.h>
 
 __u32 uk_swrandr_gen_seed32(void)
 {
+	int ret;
 	__u32 val;
 
-#ifdef CONFIG_LIBUKSWRAND_INITIALSEED_TIME
-	val = (__u32)ukplat_wall_clock();
-#endif
+	ret = ukarch_random_init();
```

Comment:
@michpappas, if `uk_early_initcall(_uk_random_init);` is also called, will there be problems? If so, should we make hardware randomness a requirement for software randomness in `Config.uk`?